platform.md

```diff
@@ -451,6 +461,60 @@ Usage:
+| `<exts>`      | `CNB_IMAGE_EXTS_DIR` | `/cnb/exts`              | Path to image extensions directory (see [Image Extensions Directory Layout](#image-extensions-directory-layout) |
+| `<group>`     | `CNB_GROUP_PATH`     | `<layers>/group.toml`    | Path to group definition (see [`group.toml`](#grouptoml-toml))                                                  |
+| `<log-level>` | `CNB_LOG_LEVEL`      | `info`                   | Log Level                                                                                                       |
+| `<output>`    | `CNB_EXT_OUTPUT_DIR` | `<layers>/exts`          | Path to output directory                                                                                        |
+| `<plan>`      | `CNB_PLAN_PATH`      | `<layers>/plan.toml`     | Path to resolved build plan (see [`plan.toml`](#plantoml-toml))                                                 |
+| `<platform>`  | `CNB_PLATFORM_DIR`   | `/platform`              | Path to platform directory                                                                                      |
+
+##### Outputs
+| Output                                       | Description                                                                   |
+|----------------------------------------------|-------------------------------------------------------------------------------|
+| [exit status]                                | (see Exit Code table below for values)                                        |
+| `/dev/stdout`                                | Logs (info)                                                                   |
+| `/dev/stderr`                                | Logs (warnings, errors)                                                       |
+| `<output>/<image extension ID>/<Dockerfile>` | Generated Dockerfiles (see [Image Extension Specfication](image-extension.md) |
+| `<layers>/config/metadata.toml`              | Build metadata (see [`metadata.toml`](#metadatatoml-toml))                    |
+
+| Exit Code       | Result                               |
+|-----------------|--------------------------------------|
+| `0`             | Success                              |
+| `11`            | Platform API incompatibility error   |
+| `12`            | Buildpack API incompatibility error  |
+| `1-10`, `13-19` | Generic lifecycle errors             |
+| `91`            | Image extension build error          |
+| `90`, `92-99`   | Generation-specific lifecycle errors |
+
+- The lifecycle SHALL execute all image extensions in the order defined in `<group>` according to the process outlined in the [Buildpack Interface Specification](buildpack.md).
+- The lifecycle SHALL replace the `run-image` reference in `<analyzed>` with the selected run image referece.
+- The lifecycle SHALL add all invoked image extensions to`<layers>/config/metadata.toml`.
+- The lifecycle SHALL record all Dockerfiles in `<extender>`.
```

Comment:
This is kind of a lot of output files for the generator... I wonder how this information could be combined. It would be nice to keep the info in extender as a separate file, because eventually the `extender` will need it.